### ISSUE TITLE
bug: Angular CDK a11y fails to find element with invalid selector

### ISSUE THREAD

**Issue by N1XUS** [SAP/fundamental-ngx#5855]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
With 0.18 version of fundamental-styles we introduced `.fd-list__navigation-item` css selector, which resulted in invalid css selector inside object-list css file: `.fd-list__navigation-item--condensed[aria-pressed=true] -arrow`.
Angular CDK a11y tries to find such elements and fails, which results in failed tests for projects that are using  CDK a11y.
#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use the current version.)
> 0.30.1.rc.56
